src/toolkit/matl_buy_policy.cc

```diff
@@ -34,6 +34,19 @@ MatlBuyPolicy::~MatlBuyPolicy() {
     manager()->context()->UnregisterTrader(this);
 }
 
+MatlBuyPolicy& MatlBuyPolicy::ResetBehavior() {
```

Comment:
Does it makes sense to reset the capacity of the total inv tracker?

Reply:
I guess that's supposed to be managed by the agent (since the buy pol isn't aware of any other buffers but `buf_`) and only set up within the buy pol when the agent passes NULL, so maybe not?

Reply:
I think that's right.  The `TotalInvTracker` has a behavior independent of the individual `ResBuf` by design, although an archetype developer will want to make sure their relationship makes sense as they change some of these things.